lambdas/retrieve-cri-credential/src/main/java/uk/gov/di/ipv/core/retrievecricredential/RetrieveCriCredentialHandler.java

```diff
@@ -198,6 +222,17 @@ public Map<String, Object> handleRequest(Map<String, String> input, Context cont
                 | CiPostMitigationsException e) {
```

Comment:
Is it possible to get any of these exceptions before the `AuditEventUser` is defined - and will that be a problem?